test/README.md

```diff
@@ -36,13 +36,13 @@ Create a new database called 'cacophonytest'
   - `\q`
 
 Migrate the database to the latest version
```

Comment:
Actually maybe we should name the test config file testapp.js.   That would make sure that we are getting the right file.

Reply:
Yep, that's a good idea. Eventually I think we want the generation of the test config file to be automated but using a more obvious configuration file name now is a good idea.

I'll add that to this PR.